### DISCUSSION TITLE
How do I put the annotations in the list

### DISCUSSION THREAD

**Discussion by SohamTilekar** [squidfunk/mkdocs-material#6812] [Q&A] [answered]
The Annotations cannot be put in the list.
due to the markdown problem.

```python
- [rich](https://github.com/Textualize/rich) (1)
{ .annotate }

1.  :man_raising_hand: I'm an annotation! I can contain `code`, __formatted
    text__, images, ... basically anything that can be expressed in Markdown.
```
output: -
![image](https://github.com/squidfunk/mkdocs-material/assets/152298897/afaf7eda-30d8-429a-b785-e50c404b4ece)

**Comment by alexvoss** [comment, marked as answer]
The solution is to wrap the list in an HTML `div` element [as described in the docs](https://squidfunk.github.io/mkdocs-material/reference/annotations/?h=annotatio#in-everything-else).
